spec/controllers/goals_controller_spec.rb

```diff
@@ -30,4 +30,70 @@
       expect(response).to redirect_to(user_path(my_user.id))
     end
   end
+
+  describe "DELETE #destroy" do
+    it "responds with http status 200 ok" do
+      user = create(:user)
+      goal = create(:goal, user: user)
+
+      sign_in(user)
+
+      delete :destroy, format: :js, id: goal.id
+      expect(response).to have_http_status(200)
+    end
+
+    it "deletes the expected goal" do
+      user = create(:user)
+      goal = create(:goal, user: user)
+
+      sign_in(user)
+
+      delete :destroy, format: :js, id: goal.id
+      expect(Goal.count).to eq(0)
+    end
+
+    it "deletes only the expected goal" do
+      user = create(:user)
+      goal = create(:goal, user: user)
+      second_goal = create(:goal, user: user)
+
+      sign_in(user)
+
+      delete :destroy, format: :js, id: goal.id
+      expect(Goal.count).to eq(1)
+    end
+
+    it "does not delete goal belonging to another user" do
+      user = create(:user)
+      goal = create(:goal, user: user)
+      other_user = create(:user, email: "someone.else@gmail.com")
+
+      sign_in(other_user)
+
+      delete :destroy, format: :js, id: goal.id
+      expect(Goal.count).to eq(1)
+    end
+
+    xit "sets the successful flash message" do
+      user = create(:user)
+      goal = create(:goal, user: user)
+
+      sign_in(user)
+
+      delete :destroy, format: :js, id: goal.id
+      expect(page).to have_content("Goal completed successfully!")
+    end
+
+    xit "responds with http status 404 not found for unkown goal" do
```

Comment:
No. It fails with the following error:

```bash
ActiveRecord::RecordNotFound:
       Couldn't find Goal with 'id'=2 [WHERE "goals"."user_id" = ?]
     # ./app/controllers/goals_controller.rb:18:in `destroy'
     # /usr/local/rvm/gems/ruby-2.2.3@rails424/gems/devise-3.5.2/lib/devise/test_helpers.rb:19:in `block in process'
     # /usr/local/rvm/gems/ruby-2.2.3@rails424/gems/devise-3.5.2/lib/devise/test_helpers.rb:72:in `catch'
     # /usr/local/rvm/gems/ruby-2.2.3@rails424/gems/devise-3.5.2/lib/devise/test_helpers.rb:72:in `_catch_warden'
     # /usr/local/rvm/gems/ruby-2.2.3@rails424/gems/devise-3.5.2/lib/devise/test_helpers.rb:19:in `process'
     # ./spec/controllers/goals_controller_spec.rb:93:in `block (3 levels) in <top (required)>'
```

After updating `DELETE#destroy` with a `current_user`-scoped query, 
```ruby
    it "does not delete goal belonging to another user" do
      user = create(:user)
      goal = create(:goal, user: user)
      other_user = create(:user)
      NO_GOAL_EXCEPTION = "A goal with that ID does not exist in the current context"

      sign_in(other_user)

      delete :destroy, format: :js, id: goal.id
      expect(flash[:error]).to eq(NO_GOAL_EXCEPTION)
    end
```
fails with this same error.

Is there a way to have RSpec expect a line to fail?